docker/Makefile

```diff
@@ -0,0 +1,30 @@
+  
+default:  
+	echo 'Options are bootstrap, start, monitor, stop, restart, nuke, clean (cleans untagged/unnamed images)'
+
+start:  
+	export COMPOSE_PROJECT_NAME=myaegee && sudo docker-compose up -d
+  
+bootstrap: start monitor
+
+monitor:
+	sudo docker-compose logs -f omscore-bootstrap && sudo docker-compose logs -f omsevents-bootstrap
+
+stop:  
+	sudo docker-compose down
+  
+restart: stop bootstrap
+
+nuke:  
+	sudo docker-compose down -v
+  
+clean:
+	sudo docker rmi $(docker images -qf "dangling=true")
```

Comment:
That will remove all dangling images, also from completely unrelated docker projects. I don't think this should be in this makefile